roles/template_service_broker/templates/apiserver-config.yaml.j2

```diff
@@ -1,4 +1,6 @@
 kind: TemplateServiceBrokerConfig
 apiVersion: config.templateservicebroker.openshift.io/v1
 templateNamespaces:
-- openshift
+{% for namespace in openshift_template_service_broker_namespaces %}
```

Comment:
This change belongs here [1] instead. Perhaps make this into a param so that the TSB role can override it with a list

[1] https://github.com/openshift/origin/blob/master/install/templateservicebroker/apiserver-template.yaml#L16

Reply:
I'm not sure I agree. I think the original intention was not for ansible to sync a default config file from openshift but to create its own on the fly, hence the structure of this change. Constructing the configuration from multiple inputs in the template could get pretty messy. @bparees what's your opinion?

Reply:
I agree with @jim-minter

Reply:
This seems contrary to what @deads2k and I had discussed before... unless I misunderstood?

Reply:
hm.  I thought this was precisely the model @deads2k was proposing.

Reply:
The apiserver config should be modified in https://github.com/openshift/origin/blob/master/install/templateservicebroker/apiserver-config.yaml .  I think that this is the corresponding file here.  Then the resulting file is fed as the `API_SERVER_CONFIG` parameter of the template.

I remember @sdodson saying that there was some kind of yaml substitution you had already built for `master-config.yaml`.  This is the corresponding file for the TSB.

Reply:
@ewolinetz are you in agreement that this is the right approach now?  if not let's plan to chat tomorrow.

Reply:
(I guess based on your latest comment the answer is no, so let's plan to get a resolution on this tomorrow)